### DISCUSSION TITLE
Custom interiors/mappings doesn't work/load and can't add more mappings.

### DISCUSSION THREAD

**Discussion by Bullsito** [OwlGamingCommunity/V#30] [Support]
Allegedly, there are a few .ymap in the gamemode (interiors and exteriors) that are correctly loaded when you execute the Cooker and run the Boot.

The thing is, when you go their allegedly locations (and dimensions) they don't exist.

I even try to add some more .ymaps in case they were bugged but they didn't work either.

What am I missing there?

**Comment by Bullsito** [comment]
So, debugging I figure out the FileTransfer events doesn't invoke properly and that MIGHT BE cause the customs maps don't load ingame.

**Comment by breineng** [comment]
@Bullsito did you manage to solve the problem?
